staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -210,6 +211,10 @@ type KubeletConfiguration struct {
 	// Default: 10
 	// +optional
 	RegistryBurst int32 `json:"registryBurst,omitempty"`
+	// +optional
+	ImagePullCredentialsVerificationPolicy v1alphav1.ImagePullCredentialsVerificationPolicy `json:"imagePullCredentialsVerificationPolicy,omitempty"`
+	// +optional
+	PreloadedImagesVerificationAllowlist []string `json:"preloadImagesVerificationAllowlist,omitempty"`
```

Comment:
copy the godoc from the internal type here, this is what gets generated into config API docs